modules/cnf-performing-platform-verification-latency-tests.adoc

```diff
@@ -0,0 +1,763 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/cnf-low-latency-tuning.adoc
+
+:_content-type: PROCEDURE
+[id="cnf-performing-platform-verification-latency-tests_{context}"]
+= Performing latency tests for platform verification
+
+You can use the Cloud-native Network Functions (CNF) tests image to run latency tests on a CNF-enabled {product-title} cluster where all the components required for running CNF workloads are installed. The latency tests allows you to validate node tuning for your workload.
+
+The `cnf-tests` container image is available at `registry.redhat.io/openshift4/cnf-tests-rhel8:v{product-version}`.
```

Comment:
You might want to check if the inline {product-version} attribute works without the `subs="attributes+"`

Reply:
It does :)